lib/corsac_jwt.dart

```diff
@@ -133,7 +133,7 @@ class JWT {
   String get issuer => _claims['iss'];
 
   /// The audience of this token (value of standard `aud` claim).
-  String get audience => _claims['aud'];
+  List<String> get audience => _claims['aud'].cast<String>();
```

Comment:
According to [spec](https://tools.ietf.org/html/rfc7519#section-4.1.3) audience can either be a list or a single string value.

It seems that we should preserve this, which means having `audience` type as `Object` + updating dartdoc for the property to describe possible return values.

WDYT?